python/ray/worker.py

```diff
@@ -39,12 +39,14 @@
     random_string,
     thread_safe_client,
 )
+from ray.external_client import ExternalClient
```

Comment:
If we don't do this import, then `import ray` will still succeed even if `requests` is not installed. I suppose `requests` is probably pretty lightweight, so that may not be an issue, but it seems ok to defer this import until we actually instantiate the `ExternalClient`.

Reply:
Sounds good to me!

Reply:
I can create an optional set of dependencies for external client and add to the documentation.